examples/pure_hikari_basic_queue/bot.py

```diff
@@ -43,13 +47,13 @@ def __init__(self, **kwargs: Any) -> None:
 class EventHandler:
```

Comment:
On 27:50 and 29:75 flake8 dislikes there being a space before the whitespace, but black formats it this way. What should I do?